package.json

```
@@ -313,7 +313,7 @@
   },
   "dependencies": {
     "eslint-formatter-pretty": "^4.1.0",
-    "markdown-it-container": "^3.0.0"
+    "markdown-it-container": "^3.0.0",
```

sadly I think this will be a prettier error
```suggestion
    "markdown-it-container": "^3.0.0"
```